### PR TITLE
Use gofmt -s to match k8s formatting requirements

### LINKED ISSUE #280: adopt k8s code formatting requirements
they're using `gofmt -s` now, which is different than what our `make format` target does. we need to be in compliance in order to upstream.

### PR DESCRIPTION
**Pull request by tsenart**
Fixes #280

### CODE REVIEW

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (self *slaveStorage) getSlaveIds() []string {
 	self.Lock()
 	defer self.Unlock()
 	slaveIds := make([]string, 0, len(self.slaves))
-	for slaveID, _ := range self.slaves {
+	for slaveID := range self.slaves {
```
Comment (tsenart) [inline pkg/scheduler/scheduler.go]: Was this code correct? It seems we were collecting the indexes, not the values in the slice.

File: pkg/scheduler/podtask/port_mapping.go [hunk 2]
```diff
@@ -147,7 +147,7 @@ func defaultHostPortMapping(t *T, offer *mesos.Offer) ([]HostPortMapping, error)
 		err := &PortAllocationError{
 			PodId: t.Pod.Name,
 		}
-		for p, _ := range requiredPorts {
+		for p := range requiredPorts {
```
Comment (tsenart) [inline pkg/scheduler/podtask/port_mapping.go]: As below, was this code correct?

File: pkg/scheduler/scheduler.go (hunk 1 above)
Comment (sttts) [inline pkg/scheduler/scheduler.go]: Good catch.

Comment (tsenart) [inline pkg/scheduler/scheduler.go]: I'll follow up with a test and patch for these two cases.

Comment (tsenart) [inline pkg/scheduler/scheduler.go]: But I'll defer the impact analysis to @jdef.

Comment (jdef) [inline pkg/scheduler/scheduler.go]: `self.slaves` is a map, so `slaveID` should actually be an ID.

File: pkg/scheduler/podtask/port_mapping.go (hunk 2 above)
Comment (jdef) [inline pkg/scheduler/podtask/port_mapping.go]: `requiredPorts` is a map, so we're iterating over the keys.

**Comment by jdef** [comment]
LGTM :ship:

**Pull request merged**